### DISCUSSION TITLE
We need a PWA + offline support example app using the new SvelteKit

### DISCUSSION THREAD

**Discussion by AlbertMarashi** [sveltejs/kit#6154] [Ideas]
Title says it all

**Comment by ESchouten** [comment]
Agree, simply linking to [VitePWA](https://github.com/vite-pwa/sveltekit) is insufficient. The libraries support for offline working has been broken for months/ever. [Issue](https://github.com/vite-pwa/sveltekit/issues/16).

**Comment by dyn-do** [comment]
Still can't?
When Sapper and Svelte were separate before, it was possible to build HTML for offline, but with Sveltekit, even if you set static adapter and set ssr = false and build, it will not be HTML that can be opened locally.
